### ISSUE TITLE
[Feature Request] Ability to select column values in query results and copy them in a sheet

### ISSUE THREAD

**Issue by BFJonk** [TablePlus/TablePlus#1702]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0): postgress 11

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
3.1.2. build 292 

It would be really helpfull if you would be abele to right-click a number of selected rows in the query results pane and have the ability to copy just the column values, in a way that you can copy them into a (google or excel) sheet. 

You can now only choose the non ** values, the bold one is my suggestion to add : 
```
> Copy
> Copy Cell value
*> Copy column values*
> Copy all column values 
> Copy rows as 
```

NB. the `Copy all column values` copys the column values as a comma separated row, which can not be pasted into a sheet or table.

**Comment by acberthem** [comment]
I am using Table Plus 3.0.4 (304) and have the same issue here. I really need an option to copy query results and paste them into Google Sheets.

**Comment by huyphams** [comment]
Please open the TablePlus settings you will find that option in there @acberthem

**Comment by nachocab** [comment]
I agree with @BFJonk. I really miss not being able to select just a few columns from a few rows. My current workflow is to 1. copy all the rows
1. Paste them Visual Studio Code
1. Change the file text mode to TSV
1. Click on the first column I'm interested in
1. Run the "Column edit select" command from the Rainbow CSV extension
1. Select additional columns if needed and copy those

I guess another approach would be to copy the rows, paste them on a google sheet and copy the columns there. 🤷‍♂️ Anyway, it would be great to be able to do this natively in TablePlus.

**Comment by huyphams** [comment]
If you copy it on the table content, there is a column button at the bottom right, we will consider to add it to the query result as well.
<img width="837" alt="Screen Shot 2020-10-26 at 1 45 44 PM" src="https://user-images.githubusercontent.com/3654622/97141576-8f7d9180-1791-11eb-8f86-708e6e6bdf9a.png">

**Comment by nachocab** [comment]
Thanks, @huyphams. Unfortunately, that won't work because I often need to filter out on other columns that I don't want to copy. The plugin you provided is a good enough workaround for now

**Comment by huyphams** [comment]
Then you might need to right-click on the header and select hide this column @nachocab 
<img width="661" alt="Screen Shot 2021-05-09 at 9 40 53 AM" src="https://user-images.githubusercontent.com/3654622/117558830-a9861e00-b0aa-11eb-9f86-3ce5e9c2a3a5.png">
